rbc/src/main/java/com/library/rbc/controller/BookController.java

```diff
@@ -16,7 +19,10 @@ public class BookController {
   private final BookService bookService;
 
   @GetMapping
-  public List<BookDto> getAllBooks() {
-    return bookService.getAllBooks();
+  public Page<BookDto> getAllBooks(@RequestParam(name = "page", defaultValue = "0") int page,
+      @RequestParam(name = "size", defaultValue = "12") int size) {
```

Comment:
You can replace individual `@RequestParams` with `Pageable pageable` in the parameter list.
```
... getAllBooks(Pageable pageable) { ... }
```